src/aggregator/client/writer_test.go

```diff
@@ -201,7 +349,8 @@ func TestWriterWriteUntimedCounterWithFlushingZeroSizeBefore(t *testing.T) {
 	require.Equal(t, []byte{1, 2, 3, 4, 5, 6, 7}, enqueuedBuf.Bytes())
 }
 
-func TestWriterWriteUntimedCounterWithFlushingPositiveSizeBefore(t *testing.T) {
+//nolint:dupl
```

Comment:
Hmmmmmm I'll let this slide on this one, but seriously, if you're nolinting for duplicated code, refactor your tests ;)